person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_accessor :name, :age, :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    true if age >= 18 && @parent_permission
+  end
+
+  private
+
+  def of_age?
+    true if age >= 18
+    false
+  end
```

Comment:
- [ ] Kindly make use of the instance variable `@age` to access the age variable instead of using the local variable `age`.